content/02.intro.md

```diff
@@ -1,11 +1,11 @@
 
-Deep learning (DL), a subfield of machine learning (ML) implementing artificial neural networks with many layers, is increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
+Deep learning (DL), a subfield of machine learning (ML) focusing on the implementation of artificial neural networks with many layers, which are increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
 In many cases, novel biological insights have been revealed through careful evaluation of DL methods ranging from predicting protein-drug binding kinetics [@doi:10.1038/s41467-017-02388-1] to identifying the lab-of-origin of synthetic DNA [@doi:10.1038/s41467-018-05378-z].
-However, a lack of concise recommendations for biological applications of DL hamper newcomers wishing to apply state-of-the-art DL in their research.
-As DL is an active and specialized research area, detailed resources are rapidly rendered obsolete and few resources articulate general DL best practices to the scientific community broadly and the biological community specifically.
+However, the lack of concise recommendations for biological applications of DL poses a challenge for newcomers wishing to apply state-of-the-art DL in their research.
+As DL is an active and specialized research area, detailed resources are rapidly rendered obsolete, and only few resources articulate general DL best practices to the scientific community broadly and the biological community specifically.
 To address this issue, we solicited input from a community of researchers with varied biological and deep learning interests, who wrote this manuscript collaboratively using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-rules] and Manubot [@url:https://greenelab.github.io/meta-review/].
 
 In the course of our discussions, several themes became clear: the importance of understanding and applying ML fundamentals  [@doi:10.1186/s13040-017-0155-3] as a baseline for utilizing DL, the necessity for extensive model comparisons with careful evaluation, and the need for critical thought in interpreting results generated by means of DL, among others.
 Ultimately, the tips we collate range from high-level guidance to the implementation of best practices, and it is our hope that they will provide actionable, DL-specific advice for both new and experienced DL practitioners alike who would like to employ DL in biological research.
-By increasing the accessibility of DL techniques to biology, we aim to improve the overall quality and reporting of DL in the literature, enabling more researchers to apply these powerful methods to generate biological insights.
+By increasing the accessibility of DL techniques for applications in biological research, we aim to improve the overall quality and reporting of DL in the literature, enabling more researchers to utilize these state-of-the art techniques for predictive modeling for generating biological insights.
```

Comment:
I contend that "predictive modeling" paints an incomplete picture of where DL can and should be successfully applied to comp bio research. For instance, many of these assorted single cell VAE-based methods  (e.g. scVI, article here https://doi.org/10.1038/s41592-018-0229-2, and  commentary from @cgreene here https://doi.org/10.1038/s41592-018-0230-9 ) provide functionality besides prediction, such as dimensionality reduction etc.

Reply:
True, I agree. I felt like this article was at times very (although not entirely) predictive modeling-specific ... 

What about "state-of-the art techniques for predictive modeling and and pattern mining" (or shall we drop that part entirely in that sentence?)

Reply:
I would say drop the changes for the second part of the sentence, since I'm not sure what they add to this sentence. The first part looks good to me though.

Reply:
Yeah the whole "for predictive modeling for generating biological insights." part is redundant. Will remove that